apis/cluster/v1alpha1/types.go

```diff
@@ -36,6 +36,15 @@ type ClusterParameters struct {
 	// Shard contains optional shard number. Calculated on the fly by the application controller if not specified.
 	// +optional
 	Shard *int64 `json:"shard,omitempty"`
+	// Reference between project and cluster that allow you automatically to be added as item inside Destinations project entity
+	// +optional
+	Project string `json:"project,omitempty"`
```

Comment:
```suggestion
	Project *string `json:"project,omitempty"`
```

Optional fields should be a pointer

Reply:
Alright @MisterMX is it for convention ?

I noticed that the [Cluster type declared in Argocd](https://github.com/argoproj/argo-cd/blob/6da92a8e8103ce4145bb0fe2b7e952be79c9ff0a/pkg/apis/application/v1alpha1/types.go#L1258-L1289) only uses a pointer for the shard option

Reply:
[Yes, this is a k8s API convention](https://github.com/kubernetes/community/blob/c6e1e89a/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required.)